pkg/network/dhcp/dhcp.go

```diff
@@ -38,6 +60,61 @@ func (d *Probe) Start(inf string) error {
 	return nil
 }
 
+// Start runs the DHCP client process and registers it to zinit
+func (d *BackgroundProbe) Start() error {
+	serviceName := fmt.Sprintf("dhcp-%s", d.inf)
+
+	ns, err := exec.Command("ip", "netns", "identify").Output()
+	if err != nil {
+		return errors.Wrap(err, "failed to identify namespace")
+	}
+
+	exec := fmt.Sprintf("/sbin/udhcpc -v -f -i %s -t 20 -T 1 -s /usr/share/udhcp/simple.script", d.inf)
+
+	if strings.Trim(string(ns), " ") != "" {
```

Comment:
Use `strings.TrimSpaces()` instead. it will also cover utf-8 spaces, tabs and new lines

Reply:
it's actually better if you do `ns = strings.TrimSpaces(ns)` and then use the cleaned up ns because you still using `ns` in code below